frontend/ui/widget/progresswidget.lua

```diff
@@ -77,7 +77,7 @@ function ProgressWidget:paintTo(bb, x, y)
         local bar_height = my_size.h-2*(self.margin_v+self.bordersize)
         for i=1, #self.ticks do
             bb:paintRect(
-                x + bar_width*(self.ticks[i]/self.last),
+                x + bar_width*(self.ticks[i]/self.last) + self.margin_h,
```

Comment:
Standard question with margins, is this correct or should it be 2*margin?

Reply:
No, this time only 1*margin. We move ticks to right.